<launch>
  <arg name="kinect_name" default="camera"/>
  <arg name="topic_in" default="$(arg kinect_name)/depth_registered_filtered/points"/>
  <arg name="topic_out" default="$(arg kinect_name)/background_sub"/>
  <arg name="voxel_size" default="0.03"/>

  <node name="$(arg kinect_name)_background_sub" pkg="kinects_human_tracking" type="kinect_background_sub" output="screen">
    <param name="min_file" value="$(find kinects_human_tracking)/data/$(arg kinect_name)/min_bg.pcd" />
    <param name="max_file" value="$(find kinects_human_tracking)/data/$(arg kinect_name)/max_bg.pcd" />
    <param name="topic_in" value="$(arg topic_in)" />
    <param name="topic_out" value="$(arg topic_out)" />
    <param name="voxel_size" value="$(arg voxel_size)" />
  </node>
</launch>
